pkgs/applications/window-managers/i3/pystatus.nix

```diff
@@ -1,14 +1,17 @@
-{ stdenv, fetchurl, libpulseaudio, python3Packages, extraLibs ? [] }:
+{ stdenv, fetchFromGitHub, libpulseaudio, python3Packages, extraLibs ? [] }:
 
 python3Packages.buildPythonApplication rec {
   name = "${pname}-${version}";
```

Comment:
Drop this line

Reply:
Please elaborate, not sure what you mean.

fetchurl was for pypi, fetchFromGitHub for the rolling release version

Reply:
`name` is computed from `pname` and `version` for `buildPythonApplication`

Reply:
Understood, was fixed on the diff instead of the actual line :)